src/patternfly/components/DataList/data-list.scss

```diff
@@ -0,0 +1,207 @@
+  // reduce spacer values for up to 380px
+  @media screen and (max-width: 380px) {
+    --pf-c-data-list__item--PaddingX: var(--pf-global--spacer--lg);
+    --pf-c-data-list__toggle--MarginRight: var(--pf-global--spacer--sm);
+    --pf-c-data-list__check--MarginRight: var(--pf-global--spacer--md);
+    --pf-c-data-list__action--MarginLeft: var(--pf-global--spacer--sm);
+  }
+
+  list-style-type: disc;
+  background-color: var(--pf-c-data-list--BackgroundColor);
+  box-shadow: var(--pf-c-data-list--BoxShadow);
+}
+
+// li
+.pf-c-data-list__item {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: baseline;
+  padding-right: var(--pf-c-data-list__item--PaddingX);
+  padding-left: var(--pf-c-data-list__item--PaddingX);
+
+  // add border top to subsequent li's
+  & + & {
+    border-top: var(--pf-c-data-list__item--BorderWidth) solid var(--pf-c-data-list__item--BorderColor);
+  }
+
+  // left border treatment base
+  // color is transparent by default
+  &::before {
+    position: absolute;
+
+    // offset border positioning by parent border width
+    top: calc(var(--pf-c-data-list__item--BorderWidth) * -1);
+    left: 0;
+    z-index: var(--pf-c-data-list__item--before--ZIndex);
+    width: var(--pf-c-data-list__item--before--Width);
+    height: calc(100% + var(--pf-c-data-list__item--BorderWidth));
+    content: "";
+    background-color: var(--pf-c-data-list__item--before--BackgroundColor);
+    transition: var(--pf-c-data-list__item--before--Transition);
+  }
+
+  // expanded border treatment color
+  &.pf-m-expanded::before {
+    background-color: var(--pf-c-data-list__item--m-expanded--before--BackgroundColor);
+  }
+}
+
+// non-textual items
+.pf-c-data-list__toggle,
+.pf-c-data-list__check,
+.pf-c-data-list__action {
+  flex: none;
+}
+
+// Toggle
+.pf-c-data-list__toggle {
+  margin-right: var(--pf-c-data-list__toggle--MarginRight);
+  margin-left: var(--pf-c-data-list__toggle--MarginLeft);
+
+  // rotate icon
+  .pf-c-button > * {
+    pointer-events: none;
+    transition: var(--pf-c-data-list__toggle--c-button--Transition);
+    transform: var(--pf-c-data-list__toggle--c-button--Transform);
+
+    .pf-m-expanded & {
+      --pf-c-data-list__toggle--c-button--Transform: var(--pf-c-data-list__toggle--c-button--m-expanded--Transform);
+    }
+  }
+}
+
+// Checkbox
+.pf-c-data-list__check {
+  display: flex;
+  margin-right: var(--pf-c-data-list__check--MarginRight);
+}
+
+
+// Content cells
+.pf-c-data-list__cell {
+  flex: 1;
+  padding-top: var(--pf-c-data-list__cell--PaddingY);
+  padding-bottom: var(--pf-c-data-list__cell--PaddingY);
+
+  // for all subsequent .pf-c-data-list__cell's, set to full width
+  & + & {
+    flex: 1 0 100%;
+    order: 1;
+    padding-top: var(--pf-c-data-list__cell-cell--PaddingTop);
+
+    @media screen and (min-width: $pf-global--breakpoint--md) {
+      --pf-c-data-list__cell-cell--PaddingTop: var(--pf-c-data-list__cell-cell--md--PaddingTop);
```

Comment:
Should this be in the top of the file, with the other css vars?

Reply:
agree ^